crates/sui-adapter-transactional-tests/tests/child_count/count_decremented.move

```diff
@@ -10,138 +10,91 @@
 
 module test::m {
     use sui::tx_context::{Self, TxContext};
+    use sui::dynamic_object_field as dof;
```

Comment:
In the spirit of making this an example that others can copy-pasta, `dof` seems a bit too opaque at the call-site, how about `field` or `ofield`?

Reply:
`ofield` isn't bad. I use `dof` in a few spots.. so let me switch them to `ofield`